addons/website/views/snippets/s_map.xml

```diff
@@ -2,15 +2,19 @@
 <odoo>
 
 <template name="Map" id="s_map">
-    <section class="s_map pb56 pt56" data-map-type="m" data-map-zoom="12" t-att-data-map-address="' '.join(filter(None, (user_id.company_id.street, user_id.company_id.city, user_id.company_id.state_id.display_name, user_id.company_id.country_id.display_name)))">
+    <section class="s_map o_half_screen_height pb56 pt56" data-map-type="m" data-map-zoom="12" t-att-data-map-address="' '.join(filter(None, (user_id.company_id.street, user_id.company_id.city, user_id.company_id.state_id.display_name, user_id.company_id.country_id.display_name)))">
         <div class="map_container o_not_editable">
             <div class="css_non_editable_mode_hidden">
                 <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none">
                     An address must be specified for a map to be embedded
                 </div>
             </div>
-            <!-- GMap iframe will be injected in JS -->
-            <div class="s_map_color_filter"/>
+            <div class="s_map_color_filter">
+                <iframe class="s_map_embedded o_not_editable"
+                    width="100%" height="100%" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"
+                    src="https://maps.google.com/maps?q=250%20Executive%20Park%20Blvd%2C%20Suite%203400%20San%20Francisco%20California%20(US)%20United%20States&amp;t=m&amp;z=12&amp;ie=UTF8&amp;iwloc=&amp;output=embed">
+                </iframe>
```

Comment:
Not a fan of adding this `iframe` in the default content: it gets loaded as soon as we switch to edit mode.
I'd rather go to:
- add a `<t t-set="is_for_drag_and_drop"/>` in `website.snippets`
- add a `t-if="not is_for_drag_and_drop"` on the `iframe` or add a class in that situation that makes it non-rendered, but with a mechanism that removes it when starting to drag the block.

Reply:
@bso-odoo The task I should start working on again after OXP: https://github.com/odoo/odoo/pull/77460 :wink: There is the data-src on iframe in particular :wink: ... last push last year, I don't remember much. But if you see something small to extract in there... maybe we can merge it with this. If we have the time :grimacing: